SignalServiceKit/src/Devices/OWSRecordTranscriptJob.m

```diff
@@ -109,6 +109,7 @@ - (void)runWithAttachmentHandler:(void (^)(TSAttachmentStream *attachmentStream)
         return;
     }
 
+    [outgoingMessage saveWithTransaction:transaction];
```

Comment:
We now explicitly saved outgoing messages created in response to a sync transcript.